README.md

```diff
@@ -6,22 +6,24 @@
 [![REUSE status](https://api.reuse.software/badge/github.com/slint-ui/slint)](https://api.reuse.software/info/github.com/slint-ui/slint)
 [![Discussions](https://img.shields.io/github/discussions/slint-ui/slint)](https://github.com/slint-ui/slint/discussions)
 
+[#MadeWithSlint Applications Gallery](https://madewithslint.com/)
+
 Slint is a declarative GUI toolkit to build native user interfaces for desktop
-and embedded applications written in Rust, C++, or JavaScript. The name *Slint*
+and embedded applications written in Rust, C++, or JavaScript. The name _Slint_
```

Comment:
(would be nice if you could configure your tools not to do this kind of edits: make it ignore markdown files)